### PR TITLE
[Student][MBL-14606] Java -> Kotlin (Part 2)

### PR DESCRIPTION
**Pull request by JordanMarshall**

### CODE REVIEW

File: apps/student/src/main/java/com/instructure/student/activity/InternalWebViewActivity.kt [hunk 1]
```diff
@@ -0,0 +1,121 @@
...
+    override fun onUpPressed() = finish()
+
+    override fun onBackPressed() {
+        val fragment = supportFragmentManager.findFragmentById(R.id.container) as? InternalWebviewFragment
+        if (fragment?.handleBackPressed() != true) super.onBackPressed()
+    }
+
+    private fun setActionBarStatusBarColors(actionBarColor: Int, statusBarColor: Int) {
+        val colorDrawable = ColorDrawable(actionBarColor)
+        supportActionBar!!.setBackgroundDrawable(colorDrawable)
```
Comment (Blackchai) [inline apps/student/src/main/java/com/instructure/student/activity/InternalWebViewActivity.kt:75]: safe call here?

File: apps/student/src/main/java/com/instructure/student/adapter/RecipientAdapter.kt [hunk 2]
```diff
@@ -24,7 +24,5 @@ class RecipientAdapter(context: Context) : BaseRecipientAdapter(context) {
     override fun getRecipientManager() = canvasRecipientManager
 
     val canvasRecipientManager: CanvasRecipientManager
-        get() = CanvasRecipientManager.getInstance(context)
-            .setRecipientCallback(this)
-            .setPhotoCallback(this)
+        get() = CanvasRecipientManager.getInstance().setRecipientCallback(this)
```
Comment (Blackchai) [inline apps/student/src/main/java/com/instructure/student/adapter/RecipientAdapter.kt:27]: Where did the photo callback go?

Comment (JordanMarshall) [reply apps/student/src/main/java/com/instructure/student/adapter/RecipientAdapter.kt:27]: It literally did nothing, so it was removed

Comment (Blackchai) [reply apps/student/src/main/java/com/instructure/student/adapter/RecipientAdapter.kt:27]: lmao, fair nuff.

File: apps/student/src/main/java/com/instructure/student/activity/InternalWebViewActivity.kt (hunk 1 above)
Comment (Blackchai) [reply apps/student/src/main/java/com/instructure/student/activity/InternalWebViewActivity.kt:75]: On the off chance that the toolbar is null, I'd rather just have this do nothing than NPE.

Comment (JordanMarshall) [reply apps/student/src/main/java/com/instructure/student/activity/InternalWebViewActivity.kt:75]: 👍

**Pull request merged**